docs/interactions/Application_Commands.md

```diff
@@ -1,83 +1,130 @@
+| Name              | Value | Note                                    |
+| ----------------- | ----- | --------------------------------------- |
+| SUB_COMMAND       | 1     |                                         |
+| SUB_COMMAND_GROUP | 2     |                                         |
+| STRING            | 3     |                                         |
+| INTEGER           | 4     | Any integer between -2^53 and 2^53      |
+| BOOLEAN           | 5     |                                         |
+| USER              | 6     |                                         |
+| CHANNEL           | 7     | Includes all channel types + categories |
+| ROLE              | 8     |                                         |
+| MENTIONABLE       | 9     | Includes users and roles                |
+| NUMBER            | 10    | Any double between -2^53 and 2^53       |
 
-## Slash Commands and Interactions
+###### Application Command Option Choice Structure
 
-Slash Commands do not depend on a bot user in the guild, instead utilizing _only_ the [interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/) model.
+If you specify `choices` for an option, they are the **only** valid values for a user to pick
+
+| Field | Type                               | Description                                         |
+| ----- | ---------------------------------- | --------------------------------------------------- |
+| name  | string                             | 1-100 character choice name                         |
+| value | string, integer, double or boolean | value of the choice, up to 100 characters if string |
 
 ## Authorizing Your Application
 
-Since you can implement slash commands without a bot, there is a new special OAuth2 scope for applications called `applications.commands`.
+Application commands do not depend on a bot user in the guild; they use the [interactions](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/) model.
+
+Because you can implement commands without a bot, there is a new OAuth2 scope for applications called `applications.commands`.
 
 > danger
-> **In order to make Slash Commands work within a guild, the guild must authorize your application with the `applications.commands` scope. The `bot` scope is not enough.**
+> **In order to make commands work within a guild, the guild must authorize your application with the `applications.commands` scope. The `bot` scope is not enough.**
 
 When requesting this scope, we "shortcut" the OAuth2 flow similar to adding a bot. You don't need to complete the flow, exchange for a token, or any of that.
 
-If your application does not require a bot user within the guild for its commands to work, **you no longer need to add for the bot scope or specific permissions**. To clarify this point even more:
+If your application does not require a bot user within the guild for its commands to work, **you no longer need to add for the bot scope or specific permissions**.
```

Comment:
This is slightly deceiving: you may need to apply for the bot scope if you require information about the guild that is not presented in interactions, such as guild owner.

(and it seems the general solution for this, smarter interaction tokens, was declined)

Reply:
it seems correct to me; if you need to get data using a bot then the application does require a bot user in the guild